src/main/java/frc/robot/Robot.java

```diff
@@ -17,23 +17,22 @@
  */
 public class Robot extends TimedRobot {
 
-    private static final String kDefaultAuto = "Default";
-    private static final String kCustomAuto = "My Auto";
-    private String m_autoSelected;
-    private final SendableChooser<String> m_chooser = new SendableChooser<>();
-
     AprilTagHighlighter aprilTagHighlighter;
+    Spark driveLeftParent = new Spark(Constants.DRIVE_LEFT_PARENT_ID);
+    Spark driveLeftChild = new Spark(Constants.DRIVE_LEFT_CHILD_ID);
+    Spark driveRightParent = new Spark(Constants.DRIVE_RIGHT_PARENT_ID);
+    Spark driveRightChild = new Spark(Constants.DRIVE_RIGHT_CHILD_ID);
```

Comment:
We don't know which type of motor controller we will be using yet, so Sparks are placeholders for now.